database/mssql/scripts/dbscripts/PSP_PIMS_S16_00/Alter Up/01_PSP_PIMS_S15_01_Alter_Up.sql

```diff
@@ -3621,18 +3584,6 @@ GO
 IF @@ERROR <> 0 SET NOEXEC ON
 GO
 
--- Create foreign key constraint dbo.PIM_LSSTYP_PIM_LEASE_FK
```

Comment:
Same issue as last time, Aqua should not be trying to add a foreign key constraint to a code table that hasn't been populated yet. Also fixed by adding this after the population script has run.